index.js

```diff
@@ -40,6 +41,17 @@ app.post('/like', (request, response) => {
   });
 });
 
+app.get('/api/search/*', (req, res) => {
+  var results = userStore.searchUsers(req.params[0]);
+
+  console.log(chalk.green('RESULTS ARE:'));
+  console.log(results);
+  
+  res.json(results);
+});
+
+
+
 app.listen(3000, () => {
   console.log('Web Server is running on port 3000');
 });
```

Comment:
in the app.get('/api/search/*' function req.params[0] refers to the dynamic part(*) of the url so if your url is /api/search/ize then req.params[0] is 'ize'